scripts/west_commands/sign.py

```diff
@@ -111,6 +111,8 @@ def do_add_parser(self, parser_adder):
                            help='''path to the tool itself, if needed''')
         group.add_argument('-D', '--tool-data', default=None,
                            help='''path to tool data/configuration directory, if needed''')
+        group.add_argument('-c', '--use-code-partition', action='store_true', default=False,
```

Comment:
Would `--sign-code-partition` be better name ?

Or maybe even make it generic, that is:
```
--partition <partition>
```
so you would invoke: `west sign --partition code-partition`.